src/vs/editor/common/config/editorOptions.ts

```diff
@@ -5701,7 +5701,16 @@ export const EditorOptions = {
 	renderValidationDecorations: register(new EditorStringEnumOption(
 		EditorOption.renderValidationDecorations, 'renderValidationDecorations',
 		'editable' as 'editable' | 'on' | 'off',
-		['editable', 'on', 'off'] as const
+		['editable', 'on', 'off'] as const,
+		{
+			description: nls.localize('renderValidationDecorations', "Controls if the workbench will show error decorations for problems."),
```

Comment:
This is slighty fishy and something that should be revisited. Technically this layer doesn't know anything about the workbench (this is the monaco editor "product's folder"). Hence this should stay an editor only setting and not become a workbench setting